canonical/canonical.com
# Product department page build

## Done

- Built Product department page
- Removed temporary redirect 

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: https://canonical-com-792.demos.haus/careers/
- See that the overview link routes to https://canonical-com-792.demos.haus/careers/product
- Check against [copy doc](https://docs.google.com/document/d/12rAis8nzWUOun0ZEK_L-ZGz1eEMzBgfc7UhN5t9OpJM/edit)
- Test CTAs

## Issue / Card

Fixes # https://warthogs.atlassian.net/browse/WD-1870
